node/connection.js

```diff
@@ -213,6 +223,26 @@ TChannelConnection.prototype.onCallResponse = function onCallResponse(res) {
     req.responseEvent.emit(req, res);
 };
 
+TChannelConnection.prototype.onPingRequest = function onPingRequest(req) {
+    var self = this;
+    req.remoteAddr = self.remoteName;
+};
+
+TChannelConnection.prototype.onPingResponse = function onPingResponse(res) {
+    var self = this;
+    var req = self.popOutReq(res.id);
```

Comment:
Did I miss where you're putting ping reqs into `self.requests.out`?  I thought we agreed to not do any frame id correlation like we do for call req/res since there's no need for correlated callbacks, and we don't intend to make it an external api.